### ISSUE TITLE
Accessibility: No accessible name for links in "info" column in the results table.

### ISSUE THREAD

**Issue by Wildebrew** [dequelabs/axe-firefox-devtools#27]
The links in the "info" column in the search results table do not
appear to have an accessible name.
Screen readers recognize them as "?".
I would propose to use an aria-label such as "more info", supplemented by an aria-describedby reference to the link's column and row headers for a full description. This second part is not necessary per WCAG 2.4.4 since the table relationship provides sufficient context, however it would be better usability to provide the full accessible description programmatically.

**Comment by dsturley** [comment]
These links no longer exist in this context.  They are now in the details pane and full text is More Info.
